src/ex4/server/services/globalConsts/GlobalConstants.js

```diff
@@ -1,9 +1,11 @@
 const NOT_A_POKEMON = "Not a pokemon";
 const POKEMON_WITH_ID_NOT_FOUND = "Pokemon with id";
 const FAILED_TO_FETCH = "Failed to fetch pokemons";
+const FIVE_MIN = 300000
```

Comment:
Even though you gave it a name the best practice is usually to name the const as of what it is in charge of like `TTL` / `CACHE_TTL` / `TIME_DELTA`

Because in the future you might want to change the time and then you'll only change the value and not the name.

Also for more readability time is written in a calculated form so it will be easy to understand debug + a commentary message like: `const TTL = 5*60*1000 //ms`